driver_test.go

```diff
@@ -661,3 +661,366 @@ func TestExecContextDml(t *testing.T) {
 	}
 
 }
+
+func TestExecContextDdl(t *testing.T) {
+
+	// Open db.
+	ctx := context.Background()
+	db, err := sql.Open("spanner", dsn)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer db.Close()
+
+	// Run tests.
+	tests := []struct {
+		name          string
+		given         string
+		withInsert    string
+		when          string
+		thenWantError bool
+		drop          string
+	}{
+		{
+			name: "create table ok",
+			when: `CREATE TABLE TestTable (
+				A   STRING(1024),
+				B  STRING(1024),
+			)	 PRIMARY KEY (A)`,
+			drop: `DROP TABLE TestTable`,
+		},
+		{
+			given: `CREATE TABLE TestTable (
+				A   STRING(1024),
+				B  STRING(1024),
+			)	 PRIMARY KEY (A)`,
+			name: "create table name duplicate",
```

Comment:
Please put the name at the top so it's easier to find the test case name

Reply:
changed